app-laptop/tlp/tlp-9999-r1.ebuild

```diff
@@ -156,9 +155,8 @@ src_install() {
 	## init/service file(s)
 	newinitd tlp.openrc "${PN}"
 
-	## repoman false positive: COPYING
-	##  specifies which files are covered by which license
-	dodoc README AUTHORS COPYING changelog
+	## specifies which files are covered by which license
+	dodoc README AUTHORS changelog
```

Comment:
I consider ``COPYING`` a false positive here.

Usually this file contains the full license (e.g. GPL-2), but for TLP,
it merely describes which code parts are covered which license.